docs/development/contributor_guidelines.rst

```diff
@@ -33,18 +33,27 @@ Code Style
 ----------
 
 We use code linters to keep a consistent code quality and style. These linters
-also run in CI and will produce build failures. To avoid this, we suggest running
-the linters locally as you work.
+also run in CI and will produce build failures. To avoid this, we have included
+a git pre-commit hook. You can install it with the following command run at the
+root of the repository:
+
+.. code:: sh
+
+    cp git/pre-commit .git/hooks/
```

Comment:
Is it folly to recommend a symlink here? The `cp` actual will not honor modifications to the pre-commit hook.

Reply:
Good point.